### ISSUE TITLE
[TalkBack] problem while reading very long text

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#229]
```
What steps will reproduce the problem?
1. Make TalkBack focus on a long text inside a TextView

What is the expected output? What do you see instead?
It should start reading the text. Nothing happens, I just see the text that 
should be read (I have that TalkBack option enabled).

What version of the product are you using? On what operating system?
Android 4.1.2 - Talkback @ Samsung S3 in spanish.

Thank you.


```

Original issue reported on code.google.com by `sebas_bs...@hotmail.com` on 18 Apr 2013 at 10:52

**Comment by GoogleCodeExporter** [comment]
```
Can't reproduce this.  Let me know if you have a specific location where this 
is happening, but we seem to be handling long blocks of text reasonably.
```

Original comment by `caseybur...@google.com` on 21 Apr 2013 at 9:31
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
Please, give it a try with this sample text I'm reproducing the issue. I 
appreciate your help! ->
"El fiscal federal Ramiro González pidió hoy la declaración indagatoria del 
contador Leonardo Fariña y del financistas Federico Elaskar por presunto 
lavado de dinero. Fuentes judiciales informaron que el pedido fue hecho al juez 
federal Sebastián Casanello en base a reportes de operaciones sospechosas por 
20 millones de pesos que presentó en la Justicia la Unidad de Información 
Financiera (UIF). Tres días después de la denuncia del programa Periodismo 
para Todos, se designó a Casanello a cargo de la causa contra el empresario 
kirchnerista Lázaro Báez por supuesto lavado de dinero. La carátula que la 
Cámara le dio al caso es, por ahora: Báez Lázaro y otro sobre encubrimiento 
y asociación ilícita. El pedido de González abre la investigación, ya que 
sin un requerimiento de instrucción del fiscal la causa no estaba oficialmente 
iniciada. En la Procuraduría de Criminalidad Económica y Lavado de Activos 
(Proselac), que creó la jefa de los fiscales, Alejandra Gils Carbó, 
anteanoche recibieron tres reportes de operaciones sospechosos (ROS) de la 
Unidad de Información Financiera (UIF), que preside José Sbattella. Dos 
corresponden a Leonardo Fariña por la compra en efectivo de autos de alta 
gama, y la tercera, a una empresa presuntamente ligada al financista Federico 
Elaskar, dijeron fuentes del caso. Casanello es uno de los cuatro jueces 
federales nombrados por Cristina Kirchner en octubre pasado. Tiene 38 años y 
es el más joven de los magistrados de Comodoro Py. Nunca fue afiliado 
partidario y su candidatura cosechó apoyos en los más variados sectores 
políticos, desde La Cámpora hasta el macrismo. Casanello llegó así al 
Juzgado Federal 7 y heredó la mitad de las causas que tenía Oyarbide."
```

Original comment by `smur...@gmail.com` on 22 Apr 2013 at 4:44

**Comment by GoogleCodeExporter** [comment]
```
I'm using default talkback version that comes with Android 4.1.2 since the 
update ONLY reads english.. while this version works for both spanish and 
english.

I'm attaching the layout with the issue to give more insight into this.
You can see TextView id=leyenda0 has a short text and it reads fine.. the next 
TextView id=leyenda00 is the long text that doesn't read.

Thanks for your help!
```

Original comment by `smur...@gmail.com` on 22 Apr 2013 at 4:51

Attachments:
- [leyendas_grales.xml](https://storage.googleapis.com/google-code-attachments/eyes-free/issue-229/comment-3/leyendas_grales.xml)
